pyproject.toml

```diff
@@ -29,7 +30,7 @@ exclude = [".git", "__pycache__", ".tox", "build", "dist", "*.egg_info", "venv"]
 select = ["E", "W", "F", "C", "N", "R", "D", "H"]
 # Ignore W503, E501 because using black creates errors with this
 # Ignore D107 Missing docstring in __init__
-ignore = ["W503", "E501", "D107"]
+ignore = ["W503", "E501", "D107", "D411"]
```

Comment:
why did we add `D411` to the ignore list?

Reply:
I'm getting a (apparent) false positive: did not give much investigation though

Reply:
I've removed the offending thing completely.